hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -272,6 +274,15 @@ const COLUMNS = [
     isSearchable: true,
     sorter: (a: IDatanode, b: IDatanode) => a.buildDate.localeCompare(b.buildDate),
     defaultSortOrder: 'ascend' as const
+  },
+  {
+    title: 'NetworkLocation',
```

Comment:
Since there are column titles it would make sense to add space instead of using CamelCase-ish names
NetworkLocation -> Network Location or Network location.
This would be needed for all the columns.

Reply:
Noted, I will create a separate ticket to normalize the format.

Reply:
Update the PR to fix the name issue, please have a check.